docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -164,10 +164,7 @@ journal:
   ufsType: "local"
   folder: "/journal"
   size: 1Gi
-  # volumeType controls the type of journal volume.
-  # It can be "persistentVolumeClaim" or "emptyDir"
   volumeType: persistentVolumeClaim
-  # Attributes to use when the journal is persistentVolumeClaim
```

Comment:
Actually I prefer explaining a bit in the doc as well. Although we have those explanations in the `values.yaml`, I believe many people just don't read carefully.